mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -666,20 +685,20 @@ func parseInnodbStatus(str string, p *map[string]float64) {
 			continue
 		}
 		if isTransaction && strings.Index(line, "------- TRX HAS BEEN") == 0 {
-			increaseMap(p, "innodb_lock_wait_secs", "1")
+			increaseMap(p, "innodb_lock_wait_secs", record[5])
 			continue
 		}
 		if strings.Index(line, "read views open inside InnoDB") > 0 {
 			(*p)["read_views"], _ = atof(record[0])
 			continue
 		}
-		if strings.Index(line, "------- TRX HAS BEEN") == 0 {
+		if strings.Index(line, "mysql tables in use") == 0 {
```

Comment:
Is it better to check `isTransaction` for safe? Or it never appears in other section?

Reply:
you are right. fixed in 1e7708b.